syncapi/storage/postgres/presence_table.go

```diff
@@ -119,20 +120,26 @@ func (p *presenceStatements) UpsertPresence(
 	return
 }
 
-// GetPresenceForUser returns the current presence of a user.
-func (p *presenceStatements) GetPresenceForUser(
+// GetPresenceForUsers returns the current presence of a user.
```

Comment:
Needs better docs. What happens if some users have no presence rows in this table?